docker/Dockerfile

```diff
@@ -1,8 +1,12 @@
 # Build frontend
 FROM node:lts-alpine as front-build-stage
 WORKDIR /front
+
+COPY ./frontend/package*.json ./
+RUN npm install
```

Comment:
unrelated to your change, but would this benefit from being switched to `npm ci`?

Reply:
Definitely, that would avoid generating Docker images when the `package-lock.json` doesn't match the defined dependencies. I can include that change in this PR.

What we can't use here is `npm ci --production`, as the `build` script needs development dependencies like `vite` and `vue-tsc`. As this is a separate stage, installing those libraries is not increasing the final image size.